impeller/docs/faq.md

```diff
@@ -100,3 +100,113 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
+  * "Impeller is AOT mode for rendering."
+* Why did the Flutter team build Impeller?
+  * The short answer, for consistent performance. The longer answer...
+  * Flutter applications, via their use of Skia, were susceptible to the problem
+    of jank due to shader compilation. Such problems were [reported as early as
```

Comment:
define jank here, in a parens (jank in this FAQ means _____)

Reply:
I attempted a definition.